src/interactions/misc/settings/settings_autocomplete.ts

```diff
@@ -1,18 +1,10 @@
-import {
-    ActionRowBuilder,
-    ButtonBuilder,
-    ButtonStyle,
-    PermissionsBitField,
-    PermissionFlagsBits,
-} from "discord.js";
 import { Command } from "../../../types/discord";
-import { embed } from "../../../utils/discord";
 import { Locales } from "../../../locales";
 
 export default {
     name: "settings-autocomplete",
     role: "AUTOCOMPLETE",
-    run: async (interaction, serverLocale, userLocale) => {
+    run: async (interaction) => {
```

Comment:
This is what it should look like

Reply:
But also not at the same time, serverLocale and userLocale need to exist otherwise when the function is called it will throw errors